lib/Driver/Job.cpp

```diff
@@ -181,6 +193,8 @@ CommandOutput::dump() const {
   llvm::errs() << '\n';
 }
 
+Job::~Job() {
```

Comment:
Nitpick: You can write this as `= default` for the same effect.

Reply:
Ok, fixed.